Sources/BraintreeCore/BTHTTP.swift

```diff
@@ -99,11 +99,17 @@ class BTHTTP: NSObject, NSCopying, URLSessionDelegate {
 
     // MARK: - HTTP Methods
 
-    func get(_ path: String, parameters: [String: Any]? = nil, shouldCache: Bool = false, completion: @escaping RequestCompletion) {
-        if shouldCache {
-            httpRequestWithCaching(method: "GET", path: path, parameters: parameters, completion: completion)
-        } else {
-            httpRequest(method: "GET", path: path, parameters: parameters, completion: completion)
+    func get(_ path: String, parameters: Encodable? = nil, shouldCache: Bool = false, completion: @escaping RequestCompletion) {
```

Comment:
In a future PR, we can make `parameters` non-optional, since there are no `get()` calls in the SDK that don't include this value.